src/execution/__tests__/oneof-test.ts

```diff
@@ -178,6 +183,354 @@ describe('Execute: Handles OneOf Input Objects', () => {
             message:
               'Variable "$input" got invalid value { a: "abc", b: null }; Exactly one key must be specified for OneOf type "TestInputObject".',
           },
+          {
+            message:
+              'Variable "$input" got invalid value null at "input.b"; Field "b" of OneOf type "TestInputObject" must be non-null.',
+            locations: [{ line: 2, column: 16 }],
+          },
```

Comment:
Two errors should not be generated for the same value - once it's determined that there was more than one key, no further checks should be performed.

Reply:
70e5c4f91